extremeflash/__main__.py

```diff
@@ -94,7 +95,7 @@ def run():
         "--model",
         action="store",
         type=str,
-        choices={"AP3710", "AP3825"},
+        choices={"AP3710", "AP3715", "AP3825"},
```

Comment:
Is this a set{} by intention?
usually a list [] is'used here.
https://docs.python.org/3/library/argparse.html#choices

I gues it's not wrong, as it works.

Reply:
Nice spot. Let me fix this in a separate PR.